docker-compose/goloop2moonbeam/docker-compose-gochain.yml

```diff
@@ -0,0 +1,103 @@
+version: "3.9"
+x-common-args: &common-args
+    GOLOOP_IMAGE: iconloop/goloop-icon:v0.9.9
+services:
+    gochain:
+        container_name: gochain
+        build:
+            context: ./gochain
+            args: 
+                - GOCHAIN_IMAGE=goloop/gochain-icon:0.9.9
+        logging:
+            driver: none
+        volumes:
+            - ${PWD}/config:/gochain/config
+            - ${PWD}/data/gochain:/gochain/data
+        ports:
+            - 9080:9080
+        environment: 
+            - GOCHAIN_CONFIG=/gochain/config/gochain.server.json
+            - GOCHAIN_DATA=/gochain/data/iconee
+            - GOCHAIN_LOGFILE=/gochain/data/gochain.log
+            - GOCHAIN_CLEAN_DATA=false
+            - GOCHAIN_PROVISION_CONFIG=/gochain/provision/config
+            - JAVAEE_BIN=/goloop/execman/bin/execman
```

Comment:
Why would I need these two environment variables? Aren't they from goloop setup, not gochain?

Reply:
Since the ENV variables are depending on provisioning, then we should hard code it in the Dockerfile. I'm moved to the Dockerfile instead.